ilastik/applets/labeling/labelingGui.py

```diff
@@ -871,3 +890,13 @@ def allowDeleteLastLabelOnly(self, enabled):
         assume that all previous cell counts are given.
         """
         self._allowDeleteLastLabelOnly = enabled
+
+    def defTwoInitialLabels(self, enabled):
+        # in some workflows wen need always at least two labels to function correctly.
+        # setting enabled to True restricts having less than two labels
```

Comment:
you mean it forces to have at least two labels?

Reply:
Yes! I'll change the comment